enterprise/frontend/src/metabase-enterprise/audit_app/reducer.ts

```diff
@@ -0,0 +1,40 @@
+import { createReducer } from "@reduxjs/toolkit";
+import { createAsyncThunk } from "metabase/lib/redux";
+
+import { GET } from "metabase/lib/api";
+import { isAuditInfoComplete } from "./selectors";
+import type { AuditInfoState } from "./types/state";
+
+const getAuditInfo = GET("/api/ee/audit-app/user/audit-info");
```

Comment:
oh, in hindsight I think i would have liked to see this in service.js or something to keep our API definitions centralized

Reply:
Good point. I'll follow this up tomorrow